satellite/gracefulexit/endpoint.go

```diff
@@ -406,7 +406,10 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 	}
 
 	if err := group.Wait(); err != nil {
-		return rpcstatus.Error(rpcstatus.Internal, Error.Wrap(err).Error())
+		if !errs.Is(err, context.Canceled) {
```

Comment:
Should we return an rpc error for context canceled?

Reply:
I had this discussion with Ethan, but we do not need to return an error for context canceled, since we return here anyway regardless of error. If we return the error on context canceled, the TestSuccess test ends up failing since `c.Recv()` returns an error for the last call.